examples/tests/statusesTest.spec.ts

```diff
@@ -28,12 +28,12 @@ test.describe('statuses for suite/test', () => {
 
   test('Test with PASSED status', () => {
     ReportingApi.setStatusPassed();
-    expect(true).toBe(false);
+    expect(true).toBe(true);
   });
 
   test('Test with FAILED status', () => {
     ReportingApi.setStatusFailed();
-    expect(true).toBe(true);
+    expect(false).toBe(true);
```

Comment:
Why are you update these examples?
It was nice to see that our `ReportingApi` method will change the test status to another.